### ISSUE TITLE
Is it possible to set different learning rate schedule for different layer in mmdetection?

### ISSUE THREAD

**Issue by Icecream-blue-sky** [open-mmlab/mmdetection#8649]
I want to set different learning rate schedule at different layer of my network. For example, use "step lr schedule" in backbone, but use "poly lr schedule" in head. Is it possible?

**Comment by chhluo** [comment]
That's not supported. But you can add param wise learn rate setting, refer to https://github.com/open-mmlab/mmdetection/blob/3b72b12fe9b14de906d1363982b9fba05e7d47c1/configs/convnext/mask_rcnn_convnext-t_p4_w7_fpn_fp16_ms-crop_3x_coco.py#L80

**Comment by Icecream-blue-sky** [comment]
> That's not supported. But you can add param wise learn rate setting, refer to
> 
> https://github.com/open-mmlab/mmdetection/blob/3b72b12fe9b14de906d1363982b9fba05e7d47c1/configs/convnext/mask_rcnn_convnext-t_p4_w7_fpn_fp16_ms-crop_3x_coco.py#L80

Param wise learn rate setting is not enough for me. Setting different learning rate schedule at different layer can be implemented in pytorch. If I want to implement it in mmdetection, any advice to achieve it ?

**Comment by chhluo** [comment]
You can implement a custmoized hook to make it, refer to https://github.com/open-mmlab/mmcv/blob/a08517790d26f8761910cac47ce8098faac7b627/mmcv/runner/hooks/lr_updater.py#L11
